README.md

```diff
@@ -132,8 +132,19 @@ go run examples/dial/psk/main.go
   openssl s_client -dtls1_2 -connect 127.0.0.1:4444 -psk abc123 -cipher PSK-AES128-CCM8
 ```
 
+### Roadmap
+The library is used as a part of our WebRTC implementation. Please refer to that [roadmap](https://github.com/pion/webrtc/issues/9) to track our major milestones.
+
+### Community
+Pion has an active community on the [Slack](https://pion.ly/slack).
+
+Follow the [Pion Twitter](https://twitter.com/_pion) for project updates and important WebRTC news.
```

Comment:
Given the current state of Twitter, I would suggest we don't push towards that right now and leave that out of the README until that situation has calmed down.

Reply:
I fully agree👍 How about a Mastodon account for news at Fosstodon.org?

Reply:
That would be nice. Since @Sean-Der normally owns all the accounts for this stuff it's probably best if they set it up though.

Reply:
Thanks for pinging Sean. I will wait until he replies and then adjust this notice in all Pion repos..

Reply:
The Pion account has 1700 followers and it's arguably where Pion has the largest reach. I would suggest not removing our active social accounts because of personal feelings about Twitter.

I'm in favor of also adding Mastodon if there's a volunteer to maintain it.

Reply:
Thats a good point. It also reflects Sean's opinion on this as discussed via Slack